source/classes.tex

```diff
@@ -228,14 +225,13 @@
 in~\ref{class.mem}.\end{note}
 
 \pnum
-\indextext{POD struct}\indextext{POD class}\indextext{POD union}%
-A \term{POD struct}\footnote{The acronym POD stands for ``plain old data''.}
+A \defn{POD struct}\footnote{The acronym POD stands for ``plain old data''.}
 is a non-union class that is both a trivial class and a
 standard-layout class, and has no non-static data members of type non-POD struct,
 non-POD union (or array of such types). Similarly, a
-\term{POD union} is a union that is both a trivial class and a standard-layout
+\defn{POD union} is a union that is both a trivial class and a standard-layout
 class, and has no non-static data members of type non-POD struct, non-POD
-union (or array of such types). A \term{POD class} is a
+union (or array of such types). A \defn{POD class} is a
 class that is either a POD struct or a POD union.
 
 \begin{example}
```

Comment:
Do we have any rules when to use "POD~struct" vs. "POD struct" in the index? We seem to prefer the variant with the tilde = non-breaking space.  If true, the stuff above needs to become \defnx.